### DISCUSSION TITLE
libgit2 v1.1.1

### DISCUSSION THREAD

**Discussion by ethomson** [libgit2/libgit2#5932] [Announcements]
🐞 This is a bugfix release with the following changes:

* Fixes a bug where decompressing packfiles could fail in rare instances.

* Ensure worktree paths are validated in more cases.

* Builds without thread-safety (`THREADSAFE=OFF`) are supported again.

* Builds without mmap (`NO_MMAP`) are supported again.

* mbedTLS is supported in non-default locations.

* Malformed branch names or missing branches on remotes are ignored.

* Use compiler intrinsics to detect arithmetic overflows in more cases.

* The configuration cache functions properly on systems with strict alignment.

* A missing options initializer function (`git_blob_filter_options_init`) was added for `git_blob_filter_options`.

* Several documentation fixes.

🎉 Thanks to all the contributors who have helped with libgit2 v1.1.1:

* @0xdky 
* @arxanas 
* @carlosmn 
* @dtolnay 
* @ethomson 
* @ianhattendorf 
* @implausible 
* @jacquesg 
* @JoshuaS3 
* @lhchavez 
* @mamapanda 
* @mjsir911 
* @punkymaniac 
* @rbmclean 
* @staticfloat 
* @staktrace 
* @tniessen 

<hr /><em>This discussion was created from the release <a href='https://github.com/libgit2/libgit2/releases/tag/v1.1.1'>libgit2 v1.1.1</a>.</em>

**Comment by Batchyx** [comment]
Hello,

It seems the release script did not upload the .tar.gz and .zip assets for this release. Can you fix it ?

**Reply by apteryks** [reply]
Hello, thanks for the release.

I'm wondering why the "release source" artifacts have vanished?  These are stable, contrary to the dynamically generated Github provided "archives", which may mutate based on infrastructure changes on their side (as it did at least once in the past).

Thank you!

**Reply by ethomson** [reply]
Because GitHub-generated archives now have the correct data.

**Reply by apteryks** [reply]
What does "now have the correct data" mean?  Does it mean GitHub generated archives are now generated once and cached forever?  If that's not the case, the risk I've written about (dynamically generated github archives changing hash) is still present.

**Reply by apteryks** [reply]
(and an annoyance to downstream users who verify the hash against an expected version)
